client/overview/survey/emoticons.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const Emoticons = ( props ) => {
+	const { icon, rating, onClick, disabled, currentRating } = props;
+
+	const iconUrl =
+		'/wp-content/plugins/woocommerce-payments/dist/../assets/images/smilies/' +
+		icon;
+
+	const buttonCss =
+		'components-button has-icon' +
+		( rating === currentRating ? ' selected' : '' );
+
+	return (
+		<>
+			<button
+				disabled={ disabled }
+				type="button"
+				className={ buttonCss }
+				onClick={ function () {
+					onClick( rating );
```

Comment:
I recommend opting for a different name to avoid potential confusion. Perhaps we should clarify what action occurs upon clicking.